aioshelly/rpc_device/device.py

```diff
@@ -233,6 +238,22 @@ async def update_config(self) -> None:
         """Get device config from 'Shelly.GetConfig'."""
         self._config = await self.call_rpc("Shelly.GetConfig")
 
+    async def update_device_info(self) -> None:
+        """Get device config from 'Shelly.GetDeviceInfo'."""
+        self._device_info = await self.call_rpc("Shelly.GetDeviceInfo")
+
```

Comment:
The data here is identical to `/shelly` endpoint which we already fetch from the device. We use the `/shelly` since it is a generic way that doesn't need auth and works on all Shelly devices.